Include the source dir, wherever it is relative to the build dir.
The original '../..' assumes that build dir is a sub-directory of source dir,
whereas Gentoo's meson eclass puts it next to it.
(This patch will put the build directory itself into the include path
as well, though -- this should not be a problem?)
--- original/post_processing_stages/meson.build  2024-09-10 13:04:09.527586248 +0200
+++ kgwportage/post_processing_stages/meson.build.includedir      2024-09-10 13:04:29.483189835 +0200
@@ -30,7 +30,7 @@
 ])
 
 core_postproc_lib = shared_module('core-postproc', core_postproc_src,
-                                  include_directories : '../..',
+                                  include_directories : include_directories('..'),
                                   dependencies : libcamera_dep,
                                   cpp_args : cpp_arguments,
                                   install : true,
